sdml-core/src/model/definitions/classes.rs

```diff
@@ -169,6 +169,8 @@ impl TypeClassDef {
         extend_variables
             => variables, TypeVariable
     );
+
+    get_and_set!(pub span, set_span, unset_span => optional has_span, Span);
```

Comment:
these are added via the `impl_has_source_span_for` macro which implements the `HasSourceSpan` trait. This trait names the field `source_span` and results in the methods `with_source_span`, `has_source_span`, `source_span`, `set_source_span`, and `unset_source_span`.